src/App.js

```diff
@@ -1,11 +1,32 @@
-import React from 'react';
-
-function App() {
-  return (
-    <h1>
-      Hello Wors
-    </h1>
-  );
+import React, { Component } from 'react';
+import Header from './Components/Header/Header';
+import Form from './Components/Form/Form';
+import WeatherDetails from './Components/WeatherDetails/WeatherDetails';
+
+class App extends Component {
+  state = {
+    weatherData: '',
```

Comment:
if the data that is comming is some kind of an object it is better to assing initial value to a `null` not to a string. 

If initial value is a string I would expect data to come bo be string also